sql/sql_select.cc

```diff
@@ -23803,14 +23803,12 @@ join_read_always_key(JOIN_TAB *tab)
   TABLE *table= tab->table;
 
   /* Initialize the index first */
-  if (!table->file->inited)
+  if ((!table->file->inited) && (unlikely((error=
```

Comment:
According to MDEV description `!table->file->inited` should be removed, right?

Reply:
Yes, we should remove it. And wait for testing pass.

Reply:
@Olernov  Remove `!table->file->inited` block would cause error, maybe we should flattern the block instead of remove it.